Library/Formula/nload.rb

```diff
@@ -11,6 +11,11 @@ class Nload < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
 
+  # F2 Option Crash and Wrong adapter read, see http://sourceforge.net/p/nload/bugs/8/
```

Comment:
Ok, can you add that as a comment here? I think I'd like other maintainers to chime in here as we may remove this software if it's effectively unmaintained now :sob: